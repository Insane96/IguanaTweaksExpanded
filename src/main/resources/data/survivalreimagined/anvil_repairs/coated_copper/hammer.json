{
	"item_to_repair": {
		"id": "survivalreimagined:coated_copper_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 5,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 5
		}
	]
}