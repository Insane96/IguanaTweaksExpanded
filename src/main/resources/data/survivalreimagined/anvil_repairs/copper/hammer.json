{
	"item_to_repair": {
		"id": "survivalreimagined:stone_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 5
		}
	]
}