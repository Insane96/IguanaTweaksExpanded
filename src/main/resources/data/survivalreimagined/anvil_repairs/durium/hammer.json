{
	"item_to_repair": {
		"id": "survivalreimagined:durium_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 5,
			"max_repair": 0.5,
			"cost_multiplier": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:durium_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "survivalreimagined:durium_nugget"
			},
			"amount": 18
		}
	]
}