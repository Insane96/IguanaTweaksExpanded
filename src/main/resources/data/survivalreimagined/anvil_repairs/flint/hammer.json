{
	"item_to_repair": {
		"id": "survivalreimagined:stone_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 5
		}
	]
}