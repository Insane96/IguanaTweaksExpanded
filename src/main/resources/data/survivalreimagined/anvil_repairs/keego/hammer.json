{
	"item_to_repair": {
		"id": "survivalreimagined:keego_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 5,
			"max_repair": 0.5,
			"cost_multiplier": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:keego"
			},
			"amount": 2
		}
	]
}