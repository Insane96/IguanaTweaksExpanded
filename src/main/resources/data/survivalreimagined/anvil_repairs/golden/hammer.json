{
	"item_to_repair": {
		"id": "survivalreimagined:golden_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 5
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 45
		}
	]
}